src/backend/storage/file/buffile.c

```diff
@@ -1038,7 +1064,7 @@ BufFileStartCompression(BufFile *file)
 	CurrentResourceOwner = file->resowner;
 
 	file->zstd_context = zstd_alloc_context();
-	file->zstd_context->cctx = ZSTD_createCStream();
+	file->zstd_context->cctx = ZSTD_createCStream_advanced(customMem); //
```

Comment:
Extra characters at the end of the line.

Reply:
That's junk too, thanks